.ci/pipelines/build.Jenkinsfile

```diff
@@ -38,6 +38,7 @@ pipeline {
                     steps {
                         sh '.ci/setenvconfig build'
                         sh 'make -C .ci license.key TARGET=ci-release ci'
+                        sh 'make -C .ci license.key TARGET=build-operator-multiarch-image ci ENABLE_FIPS=true'
```

Comment:
Would it make sense to do a whole other step here, with a different target in `setenvconfig`?  It seems overkill when looking at this, but `setenvconfig` is used to set all env variables in all other steps, but since the `build` case block is split between `nightly` and `release` already, it seemed adding a 3rd would be ugly.

Reply:
I think the parallel build makes perfect sense.  I've updated this PR with this new parallel build phase.